package-testing/android-relay/app/src/main/java/cloud/eppo/android/sdkrelay/SocketHandler.kt

```diff
@@ -0,0 +1,32 @@
+package cloud.eppo.android.sdkrelay
+
+import io.socket.client.IO
+import io.socket.client.Socket
+import java.net.URISyntaxException
+
+object SocketHandler {
```

Comment:
does this need to be a singleton?

Reply:
nope. Was just the sample I followed. Have a nice proper class now.